spec/models/spree/order_spec.rb

```diff
@@ -303,20 +306,24 @@
       order.reload.update_distribution_charge!
     end
 
-    it "returns a hash with all 3 taxes" do
-      order.tax_adjustment_totals.size.should == 3
+    it "returns a hash with all 4 taxes" do
+      order.tax_adjustment_totals.size.should == 4
     end
 
     it "contains tax on line_item" do
-      order.tax_adjustment_totals[tax_rate10.amount].should == 4.0
+      order.tax_adjustment_totals[tax_rate10].should == 4.0
     end
 
     it "contains tax on shipping_fee" do
-      order.tax_adjustment_totals[tax_rate15.amount].should == 6.0
+      order.tax_adjustment_totals[tax_rate15].should == 6.0
     end
 
     it "contains tax on enterprise_fee" do
-      order.tax_adjustment_totals[tax_rate20.amount].should == 8.0
+      order.tax_adjustment_totals[tax_rate20].should == 8.0
+    end
+
+    it "contains tax on order adjustment" do
+      order.tax_adjustment_totals[tax_rate25].should == 10.0
```

Comment:
use the `expect` syntax here as well, please

Reply:
oh yes! pretty weird that I forgot it